cloud/gcp/gce/instance/modules.tf

```diff
@@ -0,0 +1,7 @@
+module "filter-tags" {
+  source = "github.com/claranet/terraform-signalfx-detectors.git//common/filter-tags"
+
+  filter_defaults        = "filter('gcp_tag_env', '${var.environment}') and filter('gcp_tag_sfx_monitored', 'true')"
```

Comment:
As instance _labels_ (_tags_ are a network feature in GCP domain) are not collected by SignalFx's GCP integration right now, we can rely on instance _metadata_ to filter metrics:
```suggestion
  filter_defaults        = "filter('gcp_metadata_sfx_env', '${var.environment}') and filter('gcp_metadata_sfx_monitored', 'true')"
```

# Notes
* As there's a high chance of having conflicts on the `env` metadata name, I suggest prefixing it with `sfx_` like it's done for `sfx_monitored`.

* To be available, such _metadata_ has to be both:
  - added at the [instance (or project) level](https://cloud.google.com/compute/docs/storing-retrieving-metadata#custom), e.g.:
    - `gcloud compute instances add-metadata myinstance --zone=europe-west1-c --metadata sfx_env=true`
    - `gcloud compute instances add-metadata myinstance --zone=europe-west1-c --metadata sfx_monitored=true`
  - whitelisted at the SignalFx's GCP integration level (requires [SignalFx terraform provider v4.22.0+](https://github.com/terraform-providers/terraform-provider-signalfx/blob/v4.22.0/CHANGELOG.md#4220-may-29-2020), see https://docs.signalfx.com/en/latest/integrations/google-cloud-platform.html#compute-engine-instance).

WDYT?

cc @xp-1000